app/src/main/java/com/github/se/icebreakrr/MainActivity.kt

```diff
@@ -266,7 +266,11 @@ class MainActivity : ComponentActivity() {
 
   override fun onDestroy() {
     super.onDestroy()
-    // Stop monitoring when app closed
+
+    // Stop location update when app closed
+    locationViewModel.stopLocationUpdates()
```

Comment:
why would the app stop monitoring position when closed ?

Reply:
It ensures user invisibility when the app is closed, as location updates are no longer provided. This prevent from seeing outdated location datas.